scripts/gen_packaging_stats.pl

```diff
@@ -365,13 +406,38 @@ ($name, $query_ref)
 
 	my $packagings_stats_ref = {};
 
+	# Export packaging components of all products to a CSV file
+	my $filehandle;
+	my $filename = "$www_root/data/$name.csv";
```

Comment:
Couldn't we export to a csv.gz ?

Reply:
We could.. but it's just a one off at this point, so that we have data for the report of the packaging operation.